common-pg/src/main/scala/com/socrata/pg/soql/SqlFunctions.scala

```diff
@@ -262,7 +262,7 @@ object SqlFunctions extends SqlFunctionsLocation with SqlFunctionsGeometry with
       case Some(op) =>
         foldSegments(sqlFragsAndParams._1.map(s => template.format(s)), op)
     }
-    ParametricSql(Seq(foldedSql), sqlFragsAndParams._2)
+    ParametricSql(foldedSql.split(SqlFragments.SeparatorRx).toSeq, sqlFragsAndParams._2)
```

Comment:
I think what matters is that the template in formatCall do not collide with the separator.  User inputs should be using place holders and parameters.  They don't appear in the sql.